tests/unit/disco/components/TestFooter.js

```diff
@@ -0,0 +1,26 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+
+import { FooterBase } from 'disco/components/Footer';
+import { getFakeI18nInst } from 'tests/unit/helpers';
+
+
+function shallowRender(props) {
```

Comment:
boilerplate? It's just a simple change:

```js
function shallowRender(props = {}) {
}
```
but if it works as is then no worries.